service/discovery/azure/storage.go

```diff
@@ -174,12 +174,12 @@ func (d *azureStorageDiscovery) discoverCosmosDB() ([]voc.IsCloudResource, error
 
 func (d *azureStorageDiscovery) handleCosmosDB(account *armcosmos.DatabaseAccountGetResults, activityLogging *voc.ActivityLogging, raw string) ([]voc.IsCloudResource, error) {
 	var (
-		atRestEnc           voc.IsAtRestEncryption
-		err                 error
-		list                []voc.IsCloudResource
-		publicNetworkAccess = false
+		atRestEnc voc.IsAtRestEncryption
+		err       error
+		list      []voc.IsCloudResource
 	)
 
+	// TODO(lebogg): Initialization can be removed here, it is done in discoverCosmosDBs - or vice versa.
```

Comment:
However, it does not interfere, as the initClient function checks whether the client already exists.